demo/psa/manual-end-to-end.md

```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
 
 ## Preconditions
 
-* One need to install `jq` and `curl`.
+* One need to install `GO`, `jq`,`sqilte3`,`tmux` and `curl`.
+
+Installing Go, use the below mentioned path to install go in your system.
+
+`https://go.dev/doc/install`
+
```

Comment:
These look like instructions that are targeted at Debian/Ubuntu systems.  We need to make sure this is made clear to the reader (e.g., by collating them under a `### Install on Ubuntu` sub-section.)